src/components/segmentedControl/index.tsx

```diff
@@ -170,39 +180,59 @@ const SegmentedControl = (props: SegmentedControlProps) => {
 
   const renderSegments = () =>
     _.map(segments, (_value, index) => {
+      const isLastSegment = index + 1 === segments?.length;
```

Comment:
This is more 'isLast'...

Reply:
What do you mean?

Reply:
isLast is a good name to check if the current segment is the last one, while isFirstSelected is a good name to check if the selected segment is the last one

Reply:
Oh you just wanted to pin this out. I thought this wasn't good.